opencxl/cxl/device/upstream_port_device.py

```diff
@@ -88,6 +88,7 @@ def _init_device(
         self,
         mmio_manager: MmioManager,
         config_space_manager: ConfigSpaceManager,
+        _ld_id: int = 0,
```

Comment:
Not used.

Reply:
Created wrapper class: opencxl/cxl/component/cxl_io_callback_data.py